app/views/artworks/_navbar.html.erb

```diff
@@ -14,12 +14,14 @@
           <%= link_to current_user.email, "#", class: "nav-link" %>
         </li>
         <li class="nav-item">
-          <%= link_to "Messages", "#", class: "nav-link" %>
```

Comment:
Do you suggest messaging functionality between users?

Reply:
in the beginning I write Messages but then change as a Your Booking and Your Artwork. It looks like 
<img width="1420" alt="Screenshot 2021-03-02 at 19 25 08" src="https://user-images.githubusercontent.com/74418527/109703248-25d12200-7b8d-11eb-9abf-1dc8af17e49e.png">